<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>780</width>
    <height>454</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Fantastic Beasts Protocol</string>
  </property>
  <property name="accessibleName">
   <string>descrição</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListView" name="listAnimals">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>231</width>
      <height>331</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="descriptionButton">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>10</y>
      <width>141</width>
      <height>23</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Descrição</string>
    </property>
   </widget>
   <widget class="QPushButton" name="nationButton">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>10</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="accessibleName">
     <string>Nacionalidade</string>
    </property>
    <property name="text">
     <string>Nacionalidade</string>
    </property>
   </widget>
   <widget class="QPushButton" name="riscButton">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>40</y>
      <width>141</width>
      <height>23</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="accessibleName">
     <string>Riscos</string>
    </property>
    <property name="text">
     <string>Riscos</string>
    </property>
   </widget>
   <widget class="QPushButton" name="defenseButton">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>10</y>
      <width>181</width>
      <height>23</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="accessibleName">
     <string>Medidas de Proteção</string>
    </property>
    <property name="text">
     <string>Medidas de Proteção</string>
    </property>
   </widget>
   <widget class="QPushButton" name="curioButton">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>40</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="accessibleName">
     <string>Curiosidades</string>
    </property>
    <property name="text">
     <string>Curiosidades</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>90</y>
      <width>311</width>
      <height>311</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="magicalLogo">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>40</y>
      <width>231</width>
      <height>201</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/newPrefix/department-of-magic-creatures.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="ministryLogo">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>350</y>
      <width>101</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/newPrefix/ministry-of-magic (2).png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="images\logo\source.qrc"/>
  <include location="images\logo\ministry.qrc"/>
 </resources>
 <connections/>
</ui>
